Pal/src/host/Linux-SGX/enclave_framework.c

```diff
@@ -120,9 +122,10 @@ DEFINE_LISTP(trusted_file);
 static LISTP_TYPE(trusted_file) trusted_file_list = LISTP_INIT;
 static struct spinlock trusted_file_lock = LOCK_INIT;
 static int trusted_file_indexes = 0;
+static int allow_file_creation = 0;
 
 int load_trusted_file (PAL_HANDLE file, sgx_stub_t ** stubptr,
```

Comment:
@rainfld Also, under the "campsite rule" (leave it better than you found it), would you mind adding a comment above this function as well: summarizing what it does, and its inputs and outputs?